js/validation.js

```diff
@@ -0,0 +1,19 @@
+let form = document.querySelector('form')
+let email = document.getElementById('mail').value
+let error = document.querySelector('.error')
+form.addEventListener('submit', (event)=>{
+    event.preventDefault()
+    function checkEmail() {
+        if (email.match(/[A-Z]/g)) {
+            error.innerHTML = "Email address should not contain capital letters.(Not Sent)";
+            event.preventDefault()
+        }else if(email.match(/[@]/g) == false){
+            error.innerHTML ='@ missing'
+            event.preventDefault()
+        } else {
+            error.innerHTML = ''
+            return true
+        }
+    }
+    checkEmail()
```

Comment:
- [OPTIONAL] I think it would be better if you defined the `checkEmail()` separately and not inside the event listener